2 - ListsLogicAndIterations/Code/Arrays.cs

```diff
@@ -6,11 +6,23 @@ public static class Arrays
 	{
 		public static int[] BuildIntArray(int fist, int second)
 		{
+            int[] Array = new int[] { fist, second};
```

Comment:
Just to note, variables should start with lower case and in cases like this, try to use `var`. Creates cleaner code and makes it easier to refactor.